pages/ksphere/kommander/1.1/clusters/creating-konvoy-cluster-on-prem/index.md

```diff
@@ -36,5 +36,5 @@ Paste the contents of the cluster.yaml and inventory.yaml files into the form an
 ## Related information
 
 - [Install the konvoy CLI](/ksphere/konvoy/1.5/download/)
-- [Create an on premises infrastructure provider](/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/configure-on-prem-provider/)
+- [Create an on premises infrastructure provider](/ksphere/kommander/1.1/operations/infrastructure-providers/configure-on-prem-provider/)
```

Comment:
Seems like you're mixing `[]()` and `[][]` link formats in the same page. Might want to consider aggregating them all at the bottom of the page for easier mgmt later on

Reply:
yet again, not changed in this PR 🙈

Reply:
Actually I'm not sure what you've been talking about here 🙈  - All links use `[]()`?

Reply:
oh sorry that was my bad. I thought bottom of the page was collection of `[][]` links like is done [here](https://github.com/mesosphere/dcos-docs-site/pull/3090/files#diff-0d266681e6509ca90083f12501d65937R316)

nvm this comment